applications/dashboard/controllers/class.dashboardcontroller.php

```diff
@@ -59,6 +59,23 @@ public function initialize() {
     }
 
     /**
+     * An endpoint to clear all the navigation preferences for a user.
+     *
+     * @param string $userID
+     */
+    public function clearNavigationPreferences($userID = '') {
+        if ($userID && $userID !== Gdn::session()->UserID) {
+            $this->permission('Garden.Users.Edit');
+        }
+
+        $userModel = new UserModel();
+        $userModel->clearNavigationPreferences($userID);
+
+        $this->informMessage(t('Navigation preferences cleared.'));
+        redirect('/');
```

Comment:
I'd make this redirect conditional on `DELIVERY_TYPE_ALL` so we don't bounce a menu / ajax request and miss the inform message, which never actually renders as-is. Render blank if you want that to show up.